From 53209616e1ceaa2003c4536de7eb5e4d9847631f Mon Sep 17 00:00:00 2001
From: Tinywan <756684177@qq.com>
Date: Fri, 24 Nov 2017 13:19:06 +0800
Subject: [PATCH] In hls / ngx_rtmp_hls_module.c in the first 2346th line and
 2421 lines should be removed twice the line or comment out the 2346 on a [I
 am here temporarily commented out] to prevent nginx dished out configuration
 error Nginx: [emerg] The same path is used in the following languages.

---
 hls/ngx_rtmp_hls_module.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hls/ngx_rtmp_hls_module.c b/hls/ngx_rtmp_hls_module.c
index 9f6779ca6..4a15862d3 100644
--- a/hls/ngx_rtmp_hls_module.c
+++ b/hls/ngx_rtmp_hls_module.c
@@ -2343,7 +2343,7 @@ ngx_rtmp_hls_merge_app_conf(ngx_conf_t *cf, void *parent, void *child)
     ngx_conf_merge_str_value(conf->base_url, prev->base_url, "");
     ngx_conf_merge_value(conf->granularity, prev->granularity, 0);
     ngx_conf_merge_value(conf->keys, prev->keys, 0);
-    ngx_conf_merge_str_value(conf->key_path, prev->key_path, "");
+    /* ngx_conf_merge_str_value(conf->key_path, prev->key_path, "");*/
     ngx_conf_merge_str_value(conf->key_url, prev->key_url, "");
     ngx_conf_merge_uint_value(conf->frags_per_key, prev->frags_per_key, 0);